apps/ledger-live-desktop/src/renderer/components/Web3AppWebview/helpers.ts

```diff
@@ -193,6 +209,8 @@ export function useWebviewState(
     webview.addEventListener("did-start-loading", handleDidStartLoading);
     webview.addEventListener("did-stop-loading", handleDidStopLoading);
     webview.addEventListener("dom-ready", handleDomReady);
+    webview.addEventListener("did-fail-load", handleFail);
+    webview.addEventListener("crashed", handleFail);
```

Comment:
with did-fail-load, we could have more data : https://www.electronjs.org/docs/latest/api/webview-tag#event-did-fail-load. Also I believe you need to filter on isMainFrame property, even if not really well documented... We do not want to have global error if an iframe fails. So I would handle both events with 2 separate handlers. On both events, I would like to have a specific log with the specific event fields, so the CS may get data.

Please check what is the behavior when you get a 403 error or a 500 error. Does it trigger did-fail-load on 4XX/5XX ?

Reply:
Will look into this 👌